### PR TITLE
Add robots meta tag

### PR DESCRIPTION
**Pull request by lean8086**
## Change
- Added vars `PROD_ROBOTS` and `STAGING_ROBOTS` to Travis.
- Use as `ROBOTS`
- Print a meta tag to not track if it's staging.

## Related Issue
honeypotio/honeypot#4184

### CODE REVIEW

**Comment by lean8086** [comment]
We should discuss if this is the best way. I think I also can simply do something like `if (process.env.staging) { <meta .../> }`

File: .travis.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ env:
     - URL_BASE=$([ "$PREFIX_FOLDER" != "" ] && echo "/$PREFIX_FOLDER")
     - GTM_ID=$([ "$TRAVIS_TAG" != "" ] && echo "$PROD_GTM_ID" || echo "$STAGING_GTM_ID")
     - GA_ID=$([ "$TRAVIS_TAG" != "" ] && echo "$PROD_GA_ID" || echo "$STAGING_GA_ID")
+    - ROBOTS=$([ "$TRAVIS_TAG" != "" ] && echo "$PROD_ROBOTS" || echo "$STAGING_ROBOTS")
```
Comment (kdamball) [inline .travis.yml]: If it's meant to only exist in staging - why not `ROBOTS=$([ "$TRAVIS_TAG" == "" ] &&  echo "$STAGING_ROBOTS")`

Comment (kdamball) [reply .travis.yml]: maybe also a better naming here - e.g: `DISALLOW_ROBOTS`

**Comment by lean8086** [comment]
Now using only one variable on travis:
image

File: .travis.yml [hunk 2]
```diff
@@ -29,6 +29,7 @@ env:
     - URL_BASE=$([ "$PREFIX_FOLDER" != "" ] && echo "/$PREFIX_FOLDER")
     - GTM_ID=$([ "$TRAVIS_TAG" != "" ] && echo "$PROD_GTM_ID" || echo "$STAGING_GTM_ID")
     - GA_ID=$([ "$TRAVIS_TAG" != "" ] && echo "$PROD_GA_ID" || echo "$STAGING_GA_ID")
+    - DISALLOW_ROBOTS=$([ "$TRAVIS_TAG" != "" ] && echo "$STAGING_DISALLOW_ROBOTS")
```
Comment (kdamball) [inline .travis.yml]: Should be `"$TRAVIS_TAG" == ""` :) This way we know it's not a production deploy when we activate this.

**Pull request merged**